recipes/cityhash/meta.yaml

```diff
@@ -0,0 +1,42 @@
+
+{% set name = "python-cityhash" %}
+{% set version = "0.2.3.post9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/c/cityhash/cityhash-{{ version }}.tar.gz
+  sha256: aa8919c00e8d8699df42dd788eda4f24ad13086ea9d83407fd7da4a0206d9d18
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cython
```

Comment:
Cython should be a host dep

Reply:
Placed in the suggestion below: https://github.com/conda-forge/staged-recipes/pull/11057/files#r475819564